<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QMainWindow" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="w3_labelName">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>First Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_lineName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="w3_labelSurname">
          <property name="text">
           <string>Surname</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_lineSurname</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="w3_labelAge">
          <property name="text">
           <string>Age</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_lineAge</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="w3_labelEmail">
          <property name="text">
           <string>Email</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_lineEmail</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="w3_labelPass1">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_linePass1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="w3_labelPass2">
          <property name="text">
           <string>Retype Password</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>w3_linePass2</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="w3_lineName">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="w3_lineSurname">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="w3_lineAge">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="w3_lineEmail">
          <property name="text">
           <string>ytter@mail.com</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="w3_linePass1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>123456</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="w3_linePass2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>123456</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="w3_pushBack">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="w3_pushRegister">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
